{"url": "https://api.github.com/repos/Significant-Gravitas/Auto-GPT/issues/3452", "repository_url": "https://api.github.com/repos/Significant-Gravitas/Auto-GPT", "labels_url": "https://api.github.com/repos/Significant-Gravitas/Auto-GPT/issues/3452/labels{/name}", "comments_url": "https://api.github.com/repos/Significant-Gravitas/Auto-GPT/issues/3452/comments", "events_url": "https://api.github.com/repos/Significant-Gravitas/Auto-GPT/issues/3452/events", "html_url": "https://github.com/Significant-Gravitas/Auto-GPT/issues/3452", "id": 1688013050, "node_id": "I_kwDOJKSTjM5knQj6", "number": 3452, "title": "'repository_url' vs. 'url'", "user": {"login": "TerminalGravity", "id": 15236602, "node_id": "MDQ6VXNlcjE1MjM2NjAy", "avatar_url": "https://avatars.githubusercontent.com/u/15236602?v=4", "gravatar_id": "", "url": "https://api.github.com/users/TerminalGravity", "html_url": "https://github.com/TerminalGravity", "followers_url": "https://api.github.com/users/TerminalGravity/followers", "following_url": "https://api.github.com/users/TerminalGravity/following{/other_user}", "gists_url": "https://api.github.com/users/TerminalGravity/gists{/gist_id}", "starred_url": "https://api.github.com/users/TerminalGravity/starred{/owner}{/repo}", "subscriptions_url": "https://api.github.com/users/TerminalGravity/subscriptions", "organizations_url": "https://api.github.com/users/TerminalGravity/orgs", "repos_url": "https://api.github.com/users/TerminalGravity/repos", "events_url": "https://api.github.com/users/TerminalGravity/events{/privacy}", "received_events_url": "https://api.github.com/users/TerminalGravity/received_events", "type": "User", "site_admin": false}, "labels": [{"id": 5272676193, "node_id": "LA_kwDOJKSTjM8AAAABOkanYQ", "url": "https://api.github.com/repos/Significant-Gravitas/Auto-GPT/labels/bug", "name": "bug", "color": "d73a4a", "default": true, "description": "Something isn't working"}, {"id": 5380076298, "node_id": "LA_kwDOJKSTjM8AAAABQK1zCg", "url": "https://api.github.com/repos/Significant-Gravitas/Auto-GPT/labels/AI%20model%20limitation", "name": "AI model limitation", "color": "c5def5", "default": false, "description": "Not related to AutoGPT directly."}], "state": "open", "locked": false, "assignee": null, "assignees": [], "milestone": null, "comments": 2, "created_at": "2023-04-28T07:00:28Z", "updated_at": "2023-04-29T23:22:57Z", "closed_at": null, "author_association": "NONE", "active_lock_reason": null, "body": "### \u26a0\ufe0f Search for existing issues first \u26a0\ufe0f\n\n- [X] I have searched the existing issues, and there is no existing issue for my problem\n\n### Which Operating System are you using?\n\nWindows\n\n### Which version of Auto-GPT are you using?\n\nLatest Release\n\n### GPT-3 or GPT-4?\n\nGPT-4\n\n### Steps to reproduce \ud83d\udd79\n\nFollow the conversation and observe the system using 'repository_url' as an argument in the clone_repository command.\n\n### Current behavior \ud83d\ude2f\n\nThe system is using the 'repository_url' argument when calling the clone_repository command, which results in an error.\n\n### Expected behavior \ud83e\udd14\n\nThe system should use the 'url' argument when calling the clone_repository command to avoid errors.\n\n### Your prompt \ud83d\udcdd\n\n```yaml\r\n# N/A -- has to do with cloning langchain with auto-gpt to create vector stores on content exceeding max token length. \r\n```\r\n\n\n### Your Logs \ud83d\udcd2\n\n```log\r\nSYSTEM:  Command clone_repository returned: Error: clone_repository() missing 1 required positional argument: 'url'\r\n```\r\n", "reactions": {"url": "https://api.github.com/repos/Significant-Gravitas/Auto-GPT/issues/3452/reactions", "total_count": 0, "+1": 0, "-1": 0, "laugh": 0, "hooray": 0, "confused": 0, "heart": 0, "rocket": 0, "eyes": 0}, "timeline_url": "https://api.github.com/repos/Significant-Gravitas/Auto-GPT/issues/3452/timeline", "performed_via_github_app": null, "state_reason": null}